mirascope/core/anthropic/stream.py

```diff
@@ -34,9 +40,67 @@ def cost(self) -> float | None:
     def _construct_message_param(
         self, tool_calls: list[ToolUseBlock] | None = None, content: str | None = None
     ) -> MessageParam:
-        """Returns the tool message parameters for tool call results."""
+        """Constructs the message parameter for the assistant."""
+        message_param_tool_calls = []
+        if tool_calls:
+            message_param_tool_calls: list[ToolUseBlockParam] = [
+                tool_call.model_dump()
+                for tool_call in tool_calls  # type: ignore
```

Comment:
why does this need a type ignore?

Reply:
it was bc of model dump, updated to convert to ToolUseBlockParam